exercises/ayomide_bamidele/ex02_oo_basics/src/ShapeSorter.cpp

```diff
@@ -0,0 +1,50 @@
+#include "ShapeSorter.h"
+#include "Shape.h"
+#include <algorithm>
+#include <iostream>
+#include <string>
+#include <vector>
+
+int ShapeSorter::printByType(const std::vector<Shape *> &shapes,
+                             const std::string &type) {
+  for (const auto &shape : shapes) {
+    if (shape->getType() == type) {
+      std::cout << shape->getType() << " has the right type\n" << std::endl;
+    }
+  }
+  return 0;
+}
+
+int ShapeSorter::printBySide(const std::vector<Shape *> &shapes,
```

Comment:
Why do these functions need to return `int`? Why not just `void`?

Reply:
I'll implement this